src/ModelerApp.vue

```diff
@@ -61,6 +63,11 @@ export default {
   },
   data() {
     return {
+      self: this,
```

Comment:
It doesn't look like `self` is used, in which case I think it should be removed.